CHANGELOG.asciidoc

```diff
@@ -68,6 +68,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Error out on invalid Autodiscover template conditions settings. {pull}7200[7200]
 - Do not emit Kubernetes autodiscover events for Pods without IP address. {pull}7235[7235]
 - Allow to override the `ignore_above` option when defining new field with the type keyword. {pull}7238[7238]
+- Fix race condition when getting monitoring namespaces. {pull}7283[7283]
```

Comment:
If this hasn't been released we can drop the changelog

Reply:
Ok, I'll remove the line.